389.find-the-difference.js

```diff
@@ -16,5 +16,5 @@ var findTheDifference = function(s, t) {
 		if (tArr[i] !== sArr[i]) {
             return tArr[i];
         }
-	}
+    }
 };
```

Comment:
Despite the beautiful indentation on the editor, the shape is broken on Github. 
Even though I posted it again, it is not reflected in the code on Github.